packages/swapper/src/api.ts

```diff
@@ -55,18 +63,25 @@ export type CommonTradeInput = {
   wallet?: HDWallet // TODO remove this in a followup PR
 }
 
-export type GetEthTradeQuoteInput = CommonTradeInput & {
+export type GetCowTradeQuoteInput = CommonTradeInput & {
   chainId: 'eip155:1'
 }
 
+export type GetZrxTradeQuoteInput = CommonTradeInput & {
+  chainId: ZrxSupportedChainIds
+}
+
 export type GetBtcTradeQuoteInput = CommonTradeInput & {
   chainId: 'bip122:000000000019d6689c085ae165831e93'
```

Comment:
Not for this PR, but I would like to see if we could use `KnownChainIds` for the discriminator type as well

Reply:
Discriminated on `KnownChainIds` in https://github.com/shapeshift/lib/pull/850/commits/0c5e412afd95fcd6240387ff07bf805cad68d500.